lib/cypress-rails/launches_cypress.rb

```diff
@@ -20,19 +20,24 @@ def call(command, config)
         @manages_transactions.begin_transaction
       end
       server = @starts_rails_server.call(
+        host: config.host,
         port: config.port,
         transactional_server: config.transactional_server
       )
       bin = @finds_bin.call(config.dir)
 
       set_exit_hooks!(config)
 
-      command = <<~EXEC
-        CYPRESS_BASE_URL="http://#{server.host}:#{server.port}#{config.base_path}" "#{bin}" #{command} --project "#{config.dir}" #{config.cypress_cli_opts}
-      EXEC
+      if command.nil?
+        sleep
+      else
+        command = <<~EXEC
+          CYPRESS_BASE_URL="http://#{server.host}:#{server.port}#{config.base_path}" "#{bin}" #{command} --project "#{config.dir}" #{config.cypress_cli_opts}
+        EXEC
 
-      puts "\nLaunching Cypress…\n$ #{command}\n"
-      system command
+        puts "\nLaunching Cypress…\n$ #{command}\n"
+        system command
+      end
```

Comment:
@giddie is this change unrelated? Maybe if it was reverted this could get merged easier? (I'm not involved with the repo so i'm not sure, just an outside observer :D)

Reply:
This is what tripped me up about the PR too. As did the addition of a kwarg splat `**optional_args` when the `Config` object has an easy enough way to set a default for an ENV option.

I'm going to work on hand merging this with those changes and try adding a test.

Reply:
The nil check? I guess it's not related to specifying the hostname. It's certainly related to the usecase described in the pull request description, though.